protocol/protocol.proto

```diff
@@ -227,6 +227,12 @@ message SpacePayload {
         Inception inception = 1;
         // channels: sorted by channel_id
         repeated ChannelMetadata channels = 2;
+        SnappedSpaceMedia space_media = 3;
+    }
+
+    message SnappedSpaceMedia {
+        bytes creator_address = 1;
+        EncryptedData space_image = 2;
```

Comment:
can we make these names line up? I think this matches what we're doing elsewhere:
```
SnappedSpaceImage space_image = 3

message SnappedSpaceImage {
    bytes creator_address = 1;
    EncryptedData data = 2;
}
```

Reply:
looked through protocol.proto. The Snapped* and Wrapped* fields have specific names. Not a generic name like `data`

Reply:
example: 
```
 message SnappedPin {
        bytes creator_address = 1;
        Pin pin = 2;
    }
```

Reply:
WrappedEncryptedData has a data field. That is what I want to copy. 
Pins is snapshotted in a list, which has different usage syntax.

Reply:
renamed